ansible/roles/csm.cn.dns-resolver/tasks/main.yaml

```diff
@@ -77,6 +77,13 @@
       line: "NETCONFIG_DNS_RESOLVER_OPTIONS=\"{{ node_netconf_options | join(' ') }}\""
       backrefs: yes
 
+  - name: Add gcp DNS config
+    replace:
+      path: /etc/sysconfig/network/config
+      regexp: 'NETCONFIG_DNS_STATIC_SERVERS=.*'
+      replace: 'NETCONFIG_DNS_STATIC_SERVERS="10.92.100.225"'
```

Comment:
We might not want to hardcode an IP in the task. Ideally it could pull the IP with something like:
```
➜  ~ cray sls networks describe NMNLB | jq -r '.ExtraProperties.Subnets[] | .IPReservations[] | select (.Name == "unbound") | .IPAddress'
10.92.100.225
```

Of course if that IP is already defined somewhere else we could use that setting, but I'm not aware of it being defined already.

At a minimum, replace the IP with a var and set it in `csm.cn.dns-resolver/defaults`

Reply:
I don't know if we can assume that the CLI will be configured and authenticated

Reply:
True, I was intending to show the jq parsing to drill down to the dns unbound ip. Because this CFS pod is running in the services namespace, it should be able to make a request.

Here's a better example of doing something similar:
```
  - name: Get NMN Services Network info from SLS
    local_action:
      module: uri
      url: "http://cray-sls/v1/search/networks?name={{ sls_nmn_svcs_name }}"
      method: GET
    register: sls_nmn_svcs
    failed_when: sls_nmn_svcs.status != 200
    run_once: true

  - name: Get NMN Services CIDR from SLS
    set_fact:
      nmn_svcs_cidr: "{{ sls_nmn_svcs.json[0].ExtraProperties.CIDR }}"
```

Reply:
Thanks for the tip @alanm-hpe.
I updated it.